example/index.tsx

```diff
@@ -24,25 +24,38 @@ import {
 import { getTheme } from '../src/ui/theme';
 import usePublicationSW from '../src/ServiceWorker/client';
 import AxisNowEncrypted from './axisnow-encrypted';
-import htmlStyles from 'url:../src/HtmlReader/injectable-styles.css';
+import readiumBefore from 'url:../src/HtmlReader/ReadiumCss/ReadiumCSS-before.css';
+import readiumDefault from 'url:../src/HtmlReader/ReadiumCss/ReadiumCSS-default.css';
+import readiumAfter from 'url:../src/HtmlReader/ReadiumCss/ReadiumCSS-after.css';
 import Tests from './Tests';
 import { Injectable } from '../src/Readium/Injectable';
 
 const origin = window.location.origin;
 
 const pdfProxyUrl = process.env.CORS_PROXY_URL as string | undefined;
 
-const cssInjectable: Injectable = {
-  type: 'style',
-  url: htmlStyles,
-};
+const cssInjectable: Injectable[] = [
+  {
+    type: 'style',
+    url: readiumBefore,
+  },
+  {
+    type: 'style',
+    url: readiumDefault,
+  },
+  {
+    type: 'style',
+    url: readiumAfter,
+  },
+];
+
```

Comment:
Had to import these individually because of the namespace rules from the updated CSS, to get rid of the `@parcel/optimizer-css: @namespaces rules must precede all rules aside from @charset, @import, and @layer statements` error from our parcel build.

Reply:
Okay this should be fine. We'll probably need to do the same thing and make sure that each css file is published individually in the npm package however, and then we will need to update CPW and probably make a ticket to update DRB. That also means this will probably need to be a new major version